docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -152,6 +156,26 @@ To manage the media controller features in your application, you must learn to h
    mcServerInfo.sendPlaybackState('PAUSE', successCallback, errorCallback);
    ```
 
+## Setting server icon
+
+To get server icon URI from the client it has to be set by the server beforehand. Trying to read
+server icon uri before it was set will result in null value.
+
+1. Set server icon on the server side:
+
+    ```
+    var server = tizen.mediacontroller.createServer();
+    server.updateIconURI("icon.png");
+    ```
+
+2. Read server icon uri on the client side:
```

Comment:
Add “the” before server icon and change “uri” to “URI”.

Reply:
done